holo-vrrp/src/network.rs

```diff
@@ -503,6 +491,19 @@ pub(crate) async fn read_loop(
     }
 }
 
+// Helper functions.
```

Comment:
nit: use `// ===== helper functions =====` (followed by a newline) for consistency with the rest of the codebase.